src/page/utkast/skjema-section/begrunnelse/begrunnelse.tsx

```diff
@@ -84,9 +97,20 @@ function Begrunnelse() {
 							'begrunnelse__tekstomrade--feil': !!begrunnelseFeil
 						})}
 					/>
-					<Normaltekst className="begrunnelse__malform">
-						Brukers målform: {malformToTekst(malform)}
-					</Normaltekst>
+
+					<Show if={!PERSONALIA_DATA_FRA_PDL || false}>
```

Comment:
Tenker at koden for visning av malform her kan være slik som før, og at man lager en variabel ovenfor som henter malform fra v1 eller v2.

F.eks
```
const { malform, malformV2, features } =  useDataStore();
// ...
const brukersMalform = features[PERSONALIA_DATA_FRA_PDL] ? malformV2 : malform;
// ...
<Normaltekst className="begrunnelse__malform">
		Brukers målform: {malformToTekst(brukersMalform)}
</Normaltekst>
```

Reply:
Endringer er tatt vekk